spring-security-modules/spring-security-web-digest-auth/src/main/java/com/baeldung/StartApp.java

```diff
@@ -0,0 +1,13 @@
+package com.baeldung;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
```

Comment:
Maybe you missed some change in pom xml that was not committed?

Reply:
I removed that.